docs/README.md

```diff
@@ -0,0 +1,107 @@
+# Octokit Requirements
+
+> Provide a first class experience for platform developers in their preferred programming languages
+
+Octokit strives to offer complete and officially supported SDKs, quick starts, code samples, and copy/paste-able examples for the GitHub Platform in the most popular languages.
+
+Each Octokit will work towards meeting these requirements:
+
+|                                 | JavaScript | Ruby | .NET |
+| ------------------------------- | ---------- | ---- | ---- |
+| [Complete](#complete)           | 🌔         | 🌔   | 🌔   |
+| → REST                          | ✅         | ✅   | 🌔   |
+| → GraphQL                       | 🌔         | 🌑   | 🌔   |
+| → GitHub Apps                   | 🌔         | 🌔   | ✅   |
+| → OAuth                         | 🌔         | ✅   | ✅   |
+| → Webhooks                      | ✅         | 🌑   | 🌔   |
+| [Decomposable](#decomposable)   | ✅         | 🌑   | 🌑   |
+| [Generated](#generated)         | ✅         | 🌔   | 🌔   |
+| [Native](#native)               | ✅         | ✅   | ✅   |
+| [Documented](#documented)       | 🌔         | ✅   | 🌔   |
```

Comment:
> The SDK is uniformly documented, including generated API reference documentation, and getting started guides for using every platform feature.

@kytrinyx based on ☝️I'd vote to keep this as 🌔for now since we don't have getting started guides for Ruby, and the API docs are still separate from developer.github.com. That sound ok to you?